content/arduino-cloud/03.tutorials/document-projects-through-webeditor/classroom_project.md

```diff
@@ -0,0 +1,77 @@
+---
+title: 'Creating Projects using the Web Editor'
+description: 'This is the template for creating an article for the Arduino Documentation website.'
+tags: 
+  - article
+author: 'Lenard George Swamy'
+---
+
+## INTRODUCTION 
+
+Do you have an idea for your project but don’t know how to execute it for your class ? Use the web editor to start creating this project. For this example we chose to go with a simple project where you can read the distance sensor and make a proximity meter. 
+
+## HARDWARE & SOFTWARE NEEDED 
+- Arduino Uno Rev2 
+- RED LED 5mm 
+- Piezo 
+- Ultra Sonic Sense 
+- Web Editor 
+
+Note - We recommend that you read the article https://docs.arduino.cc/arduino-cloud/getting-started/getting-started-web-editor before you proceed with the tutorial. 
+
+## BUILD THE PROJECT 
+How about we start our project with getting the basics setup. Here we will run you through how the essentials elements for a project that will be essential for you to teach anything related to programming and electronics.
+
+01. Create a circuit diagram
+Start by thinking about the layout of your circuit. It could be as basic as a pen on paper but more the fidelity the easier it is for your students to understand how to connect the components. For this tutorial we created a circuit using tinkercad. 
+
+![Well thought out alternative image text.png](/Users/lenardgeorge/Documents/Arduino Drive/Cloud/Web Editor)
```

Comment:
Update the link to the asset here, it should only have to be (assets/[image title])